src/model/element-base.ts

```diff
@@ -45,13 +54,13 @@ export abstract class ScannedElementBase implements Resolvable {
     if (!this.jsdoc || !this.jsdoc.tags) {
       return;
     }
-    this.jsdoc.tags.filter((tag) => tag.tag === 'demo' && tag.name)
-        .forEach((tag) => {
-          this.demos.push({
-            desc: tag.description || undefined,
-            path: url.resolve(baseUrl, tag.name!)
-          });
-        });
+    this.demos = this.jsdoc.tags.filter((tag) => tag.tag === 'demo' && tag.name)
+                     .map((tag) => {
```

Comment:
eh, ok. IMO multi-line arrow functions should use blocks

saving a couple lines isn't worth the added difficulty in reading